src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +46,6 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        return numbers.stream().filter(number->number>3).map(number-> 2*number).reduce(0, (x,y)-> x+y);
```

Comment:
개인적으로 메서드 체이닝 사용시 가독성을 위해 한줄에 하나의 dot(.)만 사용하는게 좋다고 생각해요.
해당 람다식만 적용이 안되어있네요 :)
```suggestion
        return numbers.stream()
                .filter(number -> number > 3)
                .map(number -> 2 * number)
                .reduce(0, (x, y) -> x + y);
```